Dockerfile

```diff
@@ -11,20 +11,20 @@ RUN rm -rf /var/cache/apk/*
 WORKDIR /app
 
 # NPM
-COPY --chown=node:node package.json .
-COPY --chown=node:node package-lock.json .
-RUN npm ci --legacy-peer-deps
+COPY --chown=node:node package*.json .
+RUN npm ci --legacy-peer-deps --ignore-scripts
 RUN npm cache clean --force
 
-COPY --chown=node:node . .
+COPY --chown=node:node src src
+COPY --chown=node:node tsconfig*.json .
```

Comment:
This certainly makes code clean, but should we not be explicit with exact file names?

Reply:
in theory yes. the `*` is just a wild card to copy every file that starts with tsconfig - this is to reduce repetition. I don't mind either way tbh. let me know which one works best for him; happy to update

Reply:
Explicit file names please, thanks

Reply:
updated now